R/PlainTextParam.R

```diff
@@ -0,0 +1,215 @@
+#' @title Store `MsExperiment` and `XcmsExperiment` object as a folder of plain
+#' text files
+#'
+#' @name PlainTextParam
+#'
+#' @export
+#'
+#' @family xcms result export formats.
+#'
+#' @description
+#' The `PlainTextParam` class and method enable users to save `MsExperiment` or
+#' `XcmsExperiment` objects as collections of plain text files in a specified
+#' folder.
+#'
+#' For an `MsExperiment` object, the exported files include:
+#'
+#' - The [`sampleData()`] stored as a text file named "sample_data.txt".
+#'
+#' - The [`fileNames()`] of the *Spectra* object stored in a tabular format in a
+#' text file named "spectra_files.txt". This method only works if the `Spectra`
+#' object has an backend `MsBackendMzR`. If it does not, not spectra data will
```

Comment:
this part of the sentence is a bit odd. maybe write (instead of `This method...` and the following sentence):

```
The file names will only be exported if the `Spectra` object uses a [MsBackendMzR()] backend. 
For other backends no information on raw spectra data is currently exported with `PlainTextParam`.

```